src/libs/GetPhysicalCardUtils.ts

```diff
@@ -54,7 +54,11 @@ function setCurrentRoute(currentRoute: string, domain: string, privatePersonalDe
  * @param privatePersonalDetails
  * @returns
  */
-function getUpdatedDraftValues(draftValues: OnyxEntry<GetPhysicalCardForm>, privatePersonalDetails: OnyxEntry<PrivatePersonalDetails>, loginList: OnyxEntry<LoginList>): GetPhysicalCardForm {
+function getUpdatedDraftValues(
```

Comment:
not sure why this change would be needed, seem irrelevant?

Reply:
It was not needed, probably a merge gone wrong since I don't remember changing it.